Kickstarter-iOS/DataSources/MessageThreadsDataSource.swift

```diff
@@ -14,7 +14,7 @@ internal final class MessageThreadsDataSource: ValueCellDataSource {
   }
 
   internal func emptyState(isVisible: Bool) {
-    self.set(cellIdentifiers: isVisible ? ["MessageThreadsEmptyState"] : [],
+    self.set(cellIdentifiers: isVisible ? ["MessageThreadEmptyStateCell"] : [],
```

Comment:
yeah i could write a test. i hate loose strings and would love to have an enum for all of our cell names, but it seems like a separate task?